src/messages/Message.hpp

```diff
@@ -35,6 +35,7 @@ enum class MessageFlag : uint32_t {
     Debug = (1 << 18),
     Similar = (1 << 19),
     RedeemedHighlight = (1 << 20),
+    RedeemedChannelPointReward = (1 << 21),
```

Comment:
isn't this the same as `RedeemedHighlight` ?

Reply:
No. A channel point reward also uses a different IRC tag.